packages/core/src/components/modal/modal.scss

```diff
@@ -90,29 +90,6 @@ $modals: (
   }
```

Comment:
Could we add a `prevent` control to the story for the Modal, for us to be able to test it in storybook.